src/plugins/intel_gpu/src/graph/include/reorder_inst.h

```diff
@@ -52,7 +52,9 @@ struct typed_program_node<reorder> : public typed_program_node_base<reorder> {
             only_precision_changed &= in_layout.get_partial_shape() == out_layout.get_partial_shape();
         }
 
-        return only_precision_changed && is_simple_reorder() && typed_desc()->truncate;
+        bool is_truncate = typed_desc()->truncate || (out_layout.data_type == data_types::f32);
```

Comment:
Can you explain why this condition is needed with fp32?

Reply:
It is removed. thanks.